src/main/scala/common/ScopeMapExt.scala

```diff
@@ -0,0 +1,66 @@
+package fuselang.common
+
```

Comment:
Can you describe the changes you needed to make for `ScopeMapExt`. I'm pretty sure we can merge it with `ScopeMape`.